src/Files.App/UserControls/TabBar/TabBar.xaml

```diff
@@ -237,6 +237,7 @@
 						Grid.Column="0"
 						Width="30"
 						Height="30"
+						Margin="0,0,6,0"
```

Comment:
Was this changed on purpose?

Reply:
Add a small space to avoid overlapping elements
![image](https://github.com/user-attachments/assets/d4e79f68-43c6-4dff-9c27-538e7ff92fb2)

Reply:
There should really be a larger gap there (I think that's what the left inset is for).

Reply:
That seemed like a quick fix, if that's the problem I'd add it in a different way.

Reply:
The area next to the caption buttons is supposed to be reserved as a drag region.

Reply:
@XTorLukas I think you're experiencing this issue #14958. Can you confirm?

Reply:
Yes, it will solve the problem, but I don't think it's part of the problem of this PR, so I decided to solve it as part of issue #16532.

Reply:
Can you please remove the gap that was added in this PR as it's unrelated to these changes?

Reply:
Ok